### PR TITLE
Feature/carousel butons

### PR DESCRIPTION
**Pull request by Frankruiz90**
slider automatic, change color div
image

### CODE REVIEW

**Comment by jusk02** [comment]
Conflicts

File: index.html [hunk 1]
```diff
@@ -35,10 +35,17 @@ <h1 class="info-title">Comodidades</h1>
...
         <img src="assets/icons/arrow.svg" class="carrousel__arrow carrousel__arrow--left" data-direction="back"/>
         <img src="assets/icons/arrow.svg" class="carrousel__arrow carrousel__arrow--right" data-direction="ahead"/>
       </div>
+      <div class="pointer-list" id="divBtn">
+        <div class="pointer-list__point oval oval-gray" id="divBtn1" ></div>
+        <div class="pointer-list__point oval oval-gray" id="divBtn2" ></div>
+        <div class="pointer-list__point oval oval-gray" id="divBtn3" ></div>
+        <div class="pointer-list__point oval oval-gray" id="divBtn4" ></div>
+      </div>
+      <button href="#" class="button button--medium button--secondary">Ver galeria</button>
```
Comment (hanna446) [inline index.html:48]: I don't know if it's important, but the button component was upgraded and the classes aren't called that anymore.

File: js/index.js [hunk 2]
```diff
@@ -42,7 +50,71 @@ let currentPosition = 0;
...
+    currentPosition++;
+  } else {
+    carrouselImage.src = image[0];
+    currentPosition = 0;
+  }
+  ovalBlack
+  ovalBlack();
+};
+/*when the active page loads the interval function and sends the data and the activation
+ to the automatic function*/
```
Comment (hanna446) [inline js/index.js]: Remember the structure of comments. 
I think it's this way:
/**
*Example
*/
If  I'm wrong please correct me.

File: js/index.js [hunk 3]
```diff
@@ -42,7 +50,71 @@ let currentPosition = 0;
...
+var seg;
+window.onload = function interval() {
+   seg = setInterval(automatic,3000);
+   divCont[0].classList.add('oval-black');
+}
+
+
+
+// changes the color of the div's to black or gray
```
Comment (hanna446) [inline js/index.js]: Remember the structure of comments. 
I think it's this way:
/* Example */
If I'm wrong please correct me.

File: index.html (hunk 1 above)
Comment (jusk02) [reply index.html:48]: This will be true if the button component PR gets 2 approves before this one.

File: js/index.js (hunk 2 above)
Comment (jusk02) [reply js/index.js]: @hanna446 is right, please update the comment structure.

File: css/style.css [hunk 4]
```diff
@@ -65,7 +65,39 @@ body {
...
+  list-style-type: disc;
+}
+
+.pointer-list__point {
+  border-radius: 50%;
+  padding-left: 0%;
+  margin-left: 3%;
+  margin-right: 3%;
+  display: inline-block;
+}
```
Comment (hanna446) [inline css/style.css]: I have a suggestion.
The point distance should be a little shorter to resemble the mockup.

Comment (jusk02) [reply css/style.css]: 22px on mobile and 29px on desktop.

**Review by jhonatanrojasbastidas — APPROVED** [review]

File: css/style.css (hunk 4 above)
Comment (Frankruiz90) [reply css/style.css]: change ok.
image

File: css/style.css [hunk 5]
```diff
@@ -65,7 +65,43 @@ body {
...
+  padding-left: 0%;
+  margin-left: 14.5px;
+  margin-right: 14.5px;
+  display: inline-block;
+}
+.oval{
+  width: 14px;
+  height: 14px;
+}
+.oval-gray{
```
Comment (jusk02) [inline css/style.css]: BEM.... `ponter-list__pointer--gray` seems like the correct name, same for black

File: index.html [hunk 6]
```diff
@@ -35,10 +35,17 @@ <h1 class="info-title">Comodidades</h1>
     </div>
     <div id="carrouselContainer" class="carrousel-container">
       <div class="carrousel">
-        <img src="assets/images/IMG_8056.JPG" class="carrousel__img"/>
+        <img src="assets/images/IMG_8056.JPG" class="carrousel__img"/ id="img1">
```
Comment (jusk02) [inline index.html]: please make `id` more specific, `img1` doesnt indicate anything

File: js/index.js [hunk 7]
```diff
@@ -42,7 +50,77 @@ let currentPosition = 0;
 const carrouselArrows = document.querySelectorAll(".carrousel__arrow");
 /* Take actual image from document */
 const carrouselImage = document.querySelector(".carrousel__img");
+
+ function automatic() {
```
Comment (jusk02) [inline js/index.js]: This function could have a more indicative name like `automaticCarrousel` or `automaticChange`

File: js/index.js [hunk 8]
```diff
@@ -42,7 +50,77 @@ let currentPosition = 0;
...
+window.onload = function interval() {
+   seg = setInterval(automatic,3000);
+   divCont[0].classList.add('oval-black');
+}
+
+
+/**
+ *changes the color of the div's to black or gray
+*/
+ function ovalBlack(){
```
Comment (jusk02) [inline js/index.js]: this function could have a more indicative name like `activeOval` or `setActive`

File: js/index.js [hunk 9]
```diff
@@ -42,7 +50,76 @@ let currentPosition = 0;
...
+   currentPosition = action;
+   clearInterval(seg);
+   activeOval();
+   seg = setInterval(automaticSlider,3000);
+  }
+ };
+ /**
+  * call the html elements
+  */
+var divCont = document.getElementById('divBtn').children;
```
Comment (jusk02) [inline js/index.js]: For all variable declarations

https://github.com/airbnb/javascript#variables

**Review by jusk02 — APPROVED** [review]

**Pull request merged**